5-pod-design.md

```diff
@@ -33,7 +33,16 @@ $ kubectl run database --image=nginx --restart=Never --labels=env=prod,team=stor
 pod/database created
 ```
 
-Edit the existing Pods with the `edit` command and add the annotations as follows:
+You can assign annotations to existing pods with the `annotate` command :
+
+```bash
+$ kubectl annotate pod/frontend contact="John Doe",commit="2d3mg3" 
```

Comment:
This won't assign the annotations as expected. You'd have to separate by space instead of a comma.

Reply:
👍